docs/omero_figure_move.rst

```diff
@@ -0,0 +1,99 @@
+
+Description
+-----------
+
+This guide covers:
+
+- Manual moving of figures using user interface.
+- Retrieval of all the images contained in a Figure.
+- Duplicating of images contained in a Figure, moving them and re-creating Figure duplicates in the target group. 
+
+Setup
+-----
+
+-  Install the scripts :download:`Figure_Images_To_Dataset.py <../scripts/Figure_Images_To_Dataset.py>` and :download:`Dataset_Images_To_New_Figure.py <../scripts/Dataset_Images_To_New_Figure.py>`  on your OMERO.server.
+-  We suggest in this workflow the installation under a folder ``Figure scripts`` but you can install the script in any folder.
+-  See `server-side script guides <https://omero-guides.readthedocs.io/en/latest/scripts/docs/index.html>`__ for further details.
+
+Resources
+---------
+
+-  Any images and Figures created from these images.
+
+Step-by-Step
+------------
+
+*Moving of figures using OMERO.web user interface*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This workflow assumes that you have an OMERO.figure already created and want to move this figure into another group.
+
+#.  Log in to OMERO.web.
+
+#.  Click on ``Figure`` link above the central pane to get into OMERO.figure.
+
+#.  Inside OMERO.figure, click ``File > Open`` and open the Figure you wish to move to another group.
+
+#.  Click ``File > Move Figure to Group...``. In the following dialog, select the group you wish to move the Figure to and click OK.
+
+#.  Observe a dialog reporting a success of the move. Verify that when you now again click ``File > Move Figure to Group...`` the Figure is now reported to be in your intended group.
+
+.. note::
+      You might want to move the images contained in the Figure to the other group as well. This step is optional, if you want to use your figures by yourself only, but you should consider it in case the users you want to share your figure with do not have permissions to see your images in the original group (for example they are members of the target group only).  
+
+.. _Retrieval:
+
+*Retrieval of images contained in figures using a script*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#.  Go to OMERO.figure and open the Figure. Copy the last part of the url from the address bar of your browser, which contains the Figure ID, e.g.:
+
+    https://your-omero-server.org/web/figure/file/79828
+
+    has the Figure ID ``79828``.
+
+#.  Repeat the step above for all the Figures containing the images you want to move. Retain the Figure IDs (for example saving them as a text document) for later use.
+
+#.  Go back to the tab with OMERO.web, create a new Dataset and select it.
+
+#.  Click on the ``Scripts`` icon |image1| above the central pane of OMERO.web. Select ``Figure scripts > Figure Images to Dataset``. Start the script and paste into the dialog which appears your Figure IDs, separated with commas and no spaces, e.g. ``79828,79830,71228``. Click ``Run``.
+
```

Comment:
As above, let's focus on the workflow for a single Figure (but also mention that the script can handle multiple Figures). E.g. "Start the script and in the dialog, enter the Figure ID into the ``Figure IDs`` field. If you wish to work with multiple Figures, IDs can be separated with commas, e.g. ``79828, 79830, 71228``" (spaces are OK).

Reply:
fixed in https://github.com/ome/omero-guide-figure/pull/13/commits/4eb8f01b2bd0f64dcb0ae49158bc6592b117988c